xml/deployment_postinstall_slemicro.xml

```diff
@@ -69,30 +69,38 @@
   <title>Managing extensions</title>
 
   <para>
-   &slema; supports the kernel live patching extension. Bear in mind that the
-   extension might require an additional subscription.
+   &slema; supports the following extensions:   
   </para>
-
-  <note>
-   <title><literal>SUSE Linux Enterprise Live Patching</literal> availability</title>
+<variablelist>
+    <varlistentry>
+        <term><literal>SUSE Linux Enterprise Live Patching</literal></term>
+        <listitem>
+            <para>
+                the extension is available only for the &x86; (except for the real-time kernel) and
+    &zseries; architectures. Bear in mind that the
+   extension might require an additional subscription.
+            </para>
+        </listitem>
+    </varlistentry>
+    <varlistentry>
+        <term><literal>PackageHub</literal></term>
+        <listitem>
+            <para>
+                is a free module that provides access to community maintained packages. Packages in the Package Hub are approved by &suse; for use on &sls;, thus the packages might not be installable on &slema;.
+            </para>
+        </listitem>
+    </varlistentry>
+</variablelist>
    <para>
-    The <literal>SUSE Linux Enterprise Live Patching</literal> extension is
-    available only for the &x86; (except for the real-time kernel) and
-    &zseries; architectures.
-   </para>
-  </note>
-
-  <para>
    As the extension activation or deactivation is performed as a
    transactional-update and thus creates a new snapshot, you need to restart
```

Comment:
Do we need to mark transactional-update as a command?

Reply:
Yes.